SPECS/kernel/config

```diff
@@ -2886,7 +2886,7 @@ CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_POWERMATE is not set
```

Comment:
Since the `config` and `config_aarch64` files were changed, the hashes in kernel.signature.json need to be updated too.